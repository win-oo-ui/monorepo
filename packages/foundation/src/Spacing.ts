interface SpacingType {
  xxxs: "xxxs";
  xxs: "xxs";
  xs: "xs";
  sm: "sm";
  md: "md";
  lg: "lg";
  xl: "xl";
  xxl: "xxl";
  xxxl: "xxxl";
}

const spaces: SpacingType = {
  xxxs: "xxxs",
  xxs: "xxs",
  xs: "xs",
  sm: "sm",
  md: "md",
  lg: "lg",
  xl: "xl",
  xxl: "xxl",
  xxxl: "xxxl",
};

export default Object.freeze(spaces);
