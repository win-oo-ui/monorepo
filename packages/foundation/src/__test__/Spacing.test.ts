import Spacing from "../Spacing";

test("Snapshop of Spacings: Foundation", () => {
  expect(Spacing).toMatchSnapshot();
});
