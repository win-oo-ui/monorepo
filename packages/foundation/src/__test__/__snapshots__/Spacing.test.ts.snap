// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshop of Spacings: Foundation 1`] = `
{
  "lg": "lg",
  "md": "md",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
  "xxs": "xxs",
  "xxxl": "xxxl",
  "xxxs": "xxxs",
}
`;
